examples/superdense_coding.py

```diff
@@ -0,0 +1,63 @@
+"""Superdense Coding.
+Superdense Coding is a method to transmit two classical bits of information
+from a sender to a receiver by sending only one qubit. This is accomplished
+by pre-sharing an entangled qubit.
+
+The follwoing example sets qubit 0 to 0 and qubit 1 to 1 and are output in
+qubits 3 and 4.
+
+=== REFERENCE ===
+https://en.m.wikipedia.org/wiki/Superdense_coding
+
+=== EXAMPLE OUTPUT ===
+0: ───────────M───────@───────────────────
+                      │
+1: ───X───────M───@───┼───────────────────
+                  │   │
+2: ───H───@───────X───@───×───────────────
+          │               │
+3: ───────┼───────────────×───@───H───M───
+          │                   │
+4: ───────X───────────────────X───────M───
+
+Results:
+0=0
+1=1
+3=0
+4=1
+
+"""
+
+import cirq
+
+
+def make_superdense_circuit():
```

Comment:
Could you add comments to explain how the steps in your implementation relate to the stages in superdense coding process?